Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -416,6 +416,8 @@ enum GSError {
         let loggable = false
     }
 
+    // MARK: Delegate Key Errors
+
 	// MARK: Address Book
```

Comment:
//MARK: Delegate Key Errors should be moved closer to errors related to delegate key

Reply:
true